.gitignore

```diff
@@ -10,7 +10,6 @@
 *.pyc
 
 # Build
-vendor
```

Comment:
Undo 👍

Reply:
I did that because otherwise I had no idea vendor was messing up protoc. I can revert, but if anyone else runs `go mod vendor` they'll have problems.